sql/2020/09_Performance/performance_observer.sql

```diff
@@ -0,0 +1,23 @@
+#standardSQL
+# Percent of pages using Performance observer
+  
+SELECT
+  client,
+  COUNT(0) AS total_pages,
+  COUNTIF(performance_pages > 0) AS performance_pages,
+  COUNTIF(performance_pages > 0) / COUNT(0) AS pct_performance_pages,
+FROM (
+  SELECT
+    client,
+    COUNTIF(body LIKE '%new PerformanceObserver%') AS performance_pages,
+  FROM
+    `httparchive.almanac.summary_response_bodies`
+  WHERE
+    type = "script"
+    AND date >= "2020-09-01"
+    AND date <= "2020-09-30"
```

Comment:
I don't think this is supposed to be a timeseries.

Also, and I hate to complicate things, but we should only use the September dataset for LH performance metrics. The August dataset is the canonical one for the 2020 Web Almanac, off of which all chapters will be based. So for queries like this, we should use the August data.

```suggestion
    AND date = "2020-08-01"
```